src/rollup/types.d.ts

```diff
@@ -754,8 +754,8 @@ export interface WatcherOptions {
 	buildDelay?: number;
 	chokidar?: ChokidarOptions;
 	clearScreen?: boolean;
-	exclude?: string[];
-	include?: string[];
+	exclude?: (string | RegExp)[];
+	include?: (string | RegExp)[];
```

Comment:
Thanks for spotting! As this is directly handed to `@rollup/pluginutils`, we could actually use the type of `createFilter` here:

```suggestion
	exclude?: string | RegExp | (string | RegExp)[];
	include?: string | RegExp | (string | RegExp)[];
```